packages/semantic-model/api.d.ts

```diff
@@ -32,3 +32,13 @@ export function forEachSymbol(
   model: UI5SemanticModel,
   iteratee: (symbol: BaseUI5Node, fqn: string) => boolean | void
 ): void;
+
+/**
+ * Return a root symbol according to its fully qualified name, or undefined if not found
+ * @param model
+ * @param fqn
+ */
+export function findSymbol(
```

Comment:
How did we never need it before?

Reply:
I mean should we refactor so this utility is used in previous use cases as well?

Reply:
I think we looked for specific things before (class or namespace or enum etc) and not "any symbol with this name". Except while building the semantic model (which did use it but internally in the package it's defined).